templates/cli/lib/commands/push.js.twig

```diff
@@ -312,8 +310,8 @@ const pushFunction = async ({ functionId, all, yes, async } = {}) => {
                     func.pushVariables = true;
                 } else if (total > 0 && !func.pushVariables) {
                     log(`The function ${func.name} has remote variables setup`);
-                    const variableAnswers = await inquirer.prompt(questionsPushFunctions[1])
-                    func.pushVariables = variableAnswers.override.toLowerCase() === "yes";
+                    const variableAnswers = await inquirer.prompt(questionsPushFunctions.slice(2, 4));
+                    func.pushVariables = variableAnswers.setupoverride ? variableAnswers.setupoverride : variableAnswers.override.toLowerCase() === "yes";
```

Comment:
Same here .what is the need for the slice ?

Reply:
The same, in case the user passes the `--force` flag, then we won't ask the question, this is done like so, to make the flag control from the question instead of the code-flow itself.